Eevee.Sleep.Bot/Controllers/Mongo/MongoConst.cs

```diff
@@ -32,12 +33,21 @@ public static class MongoConst {
 
     public static readonly IMongoDatabase GameDatabase = Client.GetDatabase("game");
 
+    public static readonly IMongoCollection<OfficialSiteAnnouncementIndexModel> OfficialSiteAnnouncementIndexCollection =
```

Comment:
Oops sorry I missed this part
Make this `officialSite` (camel case) instead of `OfficialSite`, same of `inGame`